Tools/clinic/clinic.py

```diff
@@ -2820,6 +2833,14 @@ def modify(self):
         """
         return ""
 
+    def post_operations(self):
```

Comment:
The distinction between `post_operations` and `cleanup` is not immediately obvious from the name. Perhaps `post_parsing` could be an improvement?